flow/flow.proto

```diff
@@ -183,6 +195,13 @@ message Flow {
   repeated RawPacket LastRawPackets = 36;
 /* number of raw packet captured */
   int64 RawPacketsCaptured = 37;
+
+/* Flow life cycle status (Started, Updated, Ended) */
+  FlowStatus Status = 60;
```

Comment:
I'm not sure about the approach since we are adding more field for all the flows to reduce the memory used for some of them. 

Adding fields here will have memory impact and storage impact. I would rather add a unique field here
https://github.com/skydive-project/skydive/blob/master/flow/flow.go#L68 which won't be serialized, let say `terminated bool`, and in the update function check this field to generate a list and then call `ft.expireHandler.callback(expiredFlows)`

Reply:
Are you referring specifically to the field Status? or also to the other new fields FinishType & ReportCounter?

Reply:
Just one field in this structure. This structure will be not stored in the datastore but adding a field still has cost in term of memory.